Packs/IOCParser/Integrations/IOCParser/IOCParser.py

```diff
@@ -0,0 +1,417 @@
+        :rtype: ``Dict[str, Any]``
+        """
+        data = {'data': user_name}
+        return self._http_request(
+            method='POST',
+            url_suffix='/twitter',
+            headers={'Content-Type': 'application/json'},
+            data=json.dumps(data),
+            return_empty_response=True
+        )
+
+
+''' HELPER FUNCTIONS '''
+
+
+def list_to_upper_case(lst: List[str]) -> List[str]:
+    """
+    Upper case every string in the list
+
+    :type lst: ``List``
+    :param lst: The list we want to upper case
+
+    :return: The upper cased list
+    :rtype: ``List``
+    """
+    return list(map(lambda x: x.upper(), lst))
+
+
+def remove_unwanted_keys(response_data: Dict[str, List], keys: List[str]) -> None:
+    """
+    Removes all keys that were not specified by the user as the desired keys to return
+
+    :type response_data: ``Dict[str, List]``
+    :param response_data: The data key from the API's response
+
+    :type keys: ``List[str]``
+    :param keys: IOC Types to return
+    """
+
+    keys_list = list(response_data.keys())
+    for ioc_type in keys_list:
+        if ioc_type not in keys or not response_data[ioc_type]:
+            del response_data[ioc_type]
+
+
+def limit_response(response_data: Dict[str, List], limit: int) -> Dict[str, List]:
+    """
+    Trims the result from the API according to limit parameter
+
+    :type response_data: ``Dict[str, List]``
+    :param response_data: The data key from the API's response
+
+    :type limit: ``int``
+    :param limit: maximum number of results to return
+
+
+    :return: New dictionary with at most "limit" results
+    :rtype: Dict[str, List]
+    """
+
+    limited_response = {}
+    for ioc_type, iocs in response_data.items():
+        for ioc in iocs:
+            if limit > 0:
+                if not limited_response.get(ioc_type):
+                    limited_response[ioc_type] = []
+                limited_response.get(ioc_type).append(ioc)
+                limit -= 1
+    return limited_response
+
+
+def process_response(response: Dict[str, Any], keys: List[str], limit: int) -> Dict[str, List]:
+    """
+
+
+    :type response: ``Dict[str, Any]``
+    :param response: The data key from the API's response
+
+    :type keys: ``List[str]``
+    :param keys: IOC Types to return
+
+    :type limit: ``int``
+    :param limit: maximum number of results to return
+
+    :return:
+    :rtype: Dict[str, List]
+    """
+
+    try:
+        response_data = response.get('data')
+    except Exception:
+        raise Exception('The response from the API is empty')
+
+    remove_unwanted_keys(response_data, keys)
+    if limit is not None:
+        response_data = limit_response(response_data, int(limit))
+    if not response_data:
+        raise Exception('The response from the API is empty from limit')
+    return response_data
+
+
+def unite_all_tweets_into_dict(twitter_response: Dict[str, Any]) -> None:
```

Comment:
```suggestion
def unite_all_tweets_into_dict(twitter_response: Dict[str, List]) -> None:
```
According to your docstring.